tools/mo/requirements.txt

```diff
@@ -1,6 +1,8 @@
-numpy>=1.16.6
-networkx~=2.5; python_version <= "3.6"
-networkx<=2.8.8; python_version > "3.6"
-defusedxml>=0.7.1
-importlib-metadata; python_version < "3.8" and sys_platform == 'win32'
-openvino-telemetry>=2022.1.0
\ No newline at end of file
+-c ../../src/bindings/python/constraints.txt
+numpy>=1.16.6,<1.25.0
+networkx
```

Comment:
risky change to have `networkx` without upper-bound. Please recover it.

Reply:
`Add constraints.txt which allows to keep dependency versions in a single place` - FYI it is in the description. It preserves upper-bound this way.